src/frontend/js/widgets/Dashboard/utils/teacherRouteMessages.tsx

```diff
@@ -0,0 +1,158 @@
+import { defineMessages } from 'react-intl';
+
+/**
+ * All existing dashboard paths.
+ */
+export enum TeacherDashboardPaths {
+  ROOT = '/teacher',
+  TEACHER_PROFILE = '/teacher/profile',
+  TEACHER_COURSES = '/teacher/profile/courses',
+  TEACHER_SETTINGS = '/teacher/profile/settings',
+  TEACHER_NOTIFICATIONS = '/teacher/profile/notifications',
+  ORGANIZATION = '/teacher/organization/{organizationId}',
+  ORGANIZATION_COURSES = '/teacher/organization/{organizationId}/courses',
+  ORGANIZATION_SETTINGS = '/teacher/organization/{organizationId}/settings',
+  ORGANIZATION_MEMBERS = '/teacher/organization/{organizationId}/members',
+  COURSE = '/teacher/course',
+  COURSE_SETTINGS = '/teacher/course/{courseCode}/settings',
+  COURSE_CLASSROOMS = '/teacher/course/{courseCode}/classrooms',
+  COURSE_RECORDS = '/teacher/course/{courseCode}/records',
+}
+
+// Translations of dashboard route paths
+export const teacherDashboardRoutePaths = defineMessages<TeacherDashboardPaths>({
```

Comment:
General remark: I propose to rename this const to `TEACHER_DASHBOARD_ROUTE_PATHS` as it is a global constant. WDYT @all?

Reply:
+1